test/parallel/test-fs-watchfile.js

```diff
@@ -63,3 +63,25 @@ fs.watchFile(enoentFile, {interval: 0}, common.mustCall(function(curr, prev) {
     fs.unwatchFile(enoentFile);
   }
 }, 2));
+
+// Watch events should callback with a filename
+if (common.isLinux || common.isOSX) {
+  const dir = common.tmpDir + '/watch';
+
+  fs.mkdir(dir, common.mustCall(function(err) {
+    assert(!err);
+
+    fs.watch(dir, common.mustCall(function(eventType, filename) {
+      this._handle.close();
+      common.refreshTmpDir();
+      assert.strictEqual(filename, 'foo.txt');
+    }));
+
+    fs.writeFile(`${dir}/foo.txt`, 'foo', common.mustCall((err) => {
+      if (err) {
```

Comment:
you can replace the whole callback with `(err) => assert.ifError(err)`
Ref: https://github.com/nodejs/node/pull/13092#discussion_r117273934